src/xds/cluster.rs

```diff
@@ -339,13 +339,6 @@ impl ClusterManager {
         Ok(existing_endpoints)
     }
 
-    // Notify that we are about to reconnect the GRPC stream.
-    pub(in crate::xds) fn on_reconnect(&mut self) {
-        // Reset any last seen version and nonce since we'll be working
-        // with a new connection from now on with a clean slate.
-        self.last_seen_cluster_load_assignment_version = None
```

Comment:
this seems gone entirely?

Reply:
Yes, because now on failure, we just reconstruct a new `ClusterManager` altogether.

https://github.com/googleforgames/quilkin/blob/2f2667b8881e99f3ccb8ee8c0a4df8e74ac23d96/src/xds/ads_client.rs#L129-L158

Reply:
Ah got it :+1: